subsys/lorawan/lorawan.c

```diff
@@ -14,6 +15,10 @@
 #include <LoRaMac.h>
 #include <Region.h>
 
+#if !defined(CONFIG_LORAWAN_NVM_NONE)
+#include "nvm/lorawan_nvm.h"
```

Comment:
There is no real need to wrap this include in an `#if` in my opinion.

Reply:
The include is not needed if ```CONFIG_LORAWAN_NVM_NONE``` is selected.
I can remove the ```#if``` but that would add a useless ```#include``` directive